### DISCUSSION TITLE
Homebrew

### DISCUSSION THREAD

**Discussion by livid** [juicedata/juicefs#824] [General]
**What would you like to be added**:

I hope that JuiceFS can be installed via Homebrew on macOS.

```
brew install juicefs
```

**Why is this needed**:

It would make installation and update much easier for macOS users.

**Comment by yuhr123** [comment]
Hi @livid , this is supported.
```shell
$ brew tap juicedata/homebrew-tap
$ brew install juicefs
```
please check https://github.com/juicedata/juicefs/blob/main/docs/en/juicefs_on_macos.md
